torch/csrc/jit/codegen/cuda/scheduler.cpp

```diff
@@ -237,6 +238,331 @@ bool scheduleFusion(Fusion* fusion, const at::ArrayRef<c10::IValue> inputs) {
   return true;
 }
 
+namespace {
+inline int ceil_div(int a, int b) {
```

Comment:
1. inline is not needed (and won't help much, if at all, in terms of code generation)
2. naming convention: ceilDiv(), lastPow2()
3. this function may be constexpr int ceilDiv(...) - this would allow compile time evaluation if all arguments are also compile time constants

Also, while not in scope for this PR, we may want to consolidate these small helpers in a standalone header (we already have a ceilDiv() somewhere)